client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -440,16 +430,12 @@ export const ApplicationsTable: React.FC = () => {
       tableProps,
       getThProps,
       getTdProps,
-    },
-    selectionState: {
-      selectAll,
-      setSelectedItems,
-      selectedItems,
-      areAllSelected,
-      selectMultiple,
+      toolbarBulkSelectorProps,
     },
   } = tableControls;
 
+  const { selectedRows } = toolbarBulkSelectorProps;
```

Comment:
I think I would still pull `selectedItems` from `selectionState` instead of from` toolbarBulkSelectorProps` just to separate the concerns (what if we decided we don't want a bulk selector, etc)